HISTORY.md

```diff
@@ -6,13 +6,14 @@
 * Add two-phase commit support to C API.
 * Add `rocksdb_transaction_get_writebatch_wi` and `rocksdb_transaction_rebuild_from_writebatch` to C API.
 * Add SingleDelete for DB in C API
-* Add User Defined Timestamp in C API. 
+* Add User Defined Timestamp in C API.
   * `rocksdb_comparator_with_ts_create` to create timestamp aware comparator
   * Put, Get, Delete, SingleDelete, MultiGet APIs has corresponding timestamp aware APIs with suffix `with_ts`
   * And Add C API's for Transaction, SstFileWriter, Compaction as mentioned [here](https://github.com/facebook/rocksdb/wiki/User-defined-Timestamp-(Experimental))
 
 ### New Features
 * Add FileSystem::ReadAsync API in io_tracing
+* `MutableCFOptions::experimental_mempurge_threshold` can now be dynamically configured using `DB::SetOptions`.
```

Comment:
MutableCFOptions is an internal type. And moved from DBOptions to ColumnFamilyOptions?

Reply:
Thanks for correcting, I took inspiration from the "`BlockBasedTableOptions::`" option flags I saw in `HISTORY.md`. I'll replace it by `ColumnFamilyOptions:: experimental_mempurge_threshold ` and mention the move from DBOptions->ColumnFamilyOptions.